Rlabkey/R/labkey.domain.R

```diff
@@ -242,3 +242,77 @@ labkey.domain.createAndLoad <- function(baseUrl=NULL, folderPath, name, descript
 
     labkey.insertRows(baseUrl = baseUrl, folderPath = folderPath, schemaName = schemaName, queryName= name, df)
 }
+
+labkey.domain.conditionalFormat <- function(queryFilter, bold=FALSE, italic=FALSE, strikeThrough=FALSE, textColor="", backgroundColor="")
+{
+    data.frame(filter = queryFilter, bold = bold, strikethrough = strikeThrough, italic = italic, textColor = textColor, backgroundColor = backgroundColor)
+}
+
+labkey.domain.queryFilter <- function(value, filterType, additionalValue=NULL, additionalFilter=NULL)
+{
+    qf1 <- sprintf("format.column~%s=%s", filterType, value)
+    qf2 <- NULL
+
+    if (!is.null(additionalValue) || !is.null(additionalFilter))
+        qf2 <- sprintf("format.column~%s=%s", filterType, value)
+
+    qf <- if(is.null(qf2)) qf1 else paste0(qf1, "&", qf2)
+    return(qf)
+}
+
+labkey.domain.queryFilterEnum <- function()
+{
+    list(
+        HAS_ANY_VALUE = '',
```

Comment:
This casing was taken directly from the python api. Perhaps we want a different casing in R?